packages/babel-types/src/ast-types/generated/index.ts

```diff
@@ -2780,3 +2781,5773 @@ export type DeprecatedAliases =
+    | DeclareModule
+    | Decorator
+    | DoWhileStatement
+    | EnumStringMember
+    | ExportAllDeclaration
+    | ExportDefaultDeclaration
+    | ExportNamedDeclaration
+    | ExportSpecifier
+    | ExpressionStatement
+    | ForInStatement
+    | ForOfStatement
+    | ForStatement
+    | IfStatement
+    | ImportAttribute
+    | ImportDeclaration
+    | ImportSpecifier
+    | JSXAttribute
+    | JSXExpressionContainer
+    | JSXSpreadAttribute
+    | JSXSpreadChild
+    | LogicalExpression
+    | MemberExpression
+    | NewExpression
+    | ObjectMethod
+    | ObjectProperty
+    | ObjectTypeProperty
+    | OptionalCallExpression
+    | OptionalMemberExpression
+    | ParenthesizedExpression
+    | PipelineBareFunction
+    | PipelineTopicExpression
+    | ReturnStatement
+    | SequenceExpression
+    | SpreadElement
+    | SwitchCase
+    | SwitchStatement
+    | TSAsExpression
+    | TSDeclareMethod
+    | TSEnumDeclaration
+    | TSEnumMember
+    | TSExportAssignment
+    | TSExternalModuleReference
+    | TSImportType
+    | TSInstantiationExpression
+    | TSLiteralType
+    | TSMethodSignature
+    | TSModuleDeclaration
+    | TSNonNullExpression
+    | TSPropertySignature
+    | TSTypeAssertion
+    | TaggedTemplateExpression
+    | TemplateLiteral
+    | ThrowStatement
+    | TupleExpression
+    | TypeCastExpression
+    | UnaryExpression
+    | UpdateExpression
+    | VariableDeclarator
+    | WhileStatement
+    | WithStatement
+    | YieldExpression;
+  StringLiteralTypeAnnotation:
+    | ArrayTypeAnnotation
+    | DeclareExportDeclaration
+    | DeclareOpaqueType
+    | DeclareTypeAlias
+    | DeclaredPredicate
+    | FunctionTypeAnnotation
+    | FunctionTypeParam
+    | IndexedAccessType
+    | IntersectionTypeAnnotation
+    | NullableTypeAnnotation
+    | ObjectTypeCallProperty
+    | ObjectTypeIndexer
+    | ObjectTypeInternalSlot
+    | ObjectTypeProperty
+    | ObjectTypeSpreadProperty
+    | OpaqueType
+    | OptionalIndexedAccessType
+    | TupleTypeAnnotation
+    | TypeAlias
+    | TypeAnnotation
+    | TypeParameter
+    | TypeParameterInstantiation
+    | TypeofTypeAnnotation
+    | UnionTypeAnnotation;
+  StringTypeAnnotation:
+    | ArrayTypeAnnotation
+    | DeclareExportDeclaration
+    | DeclareOpaqueType
+    | DeclareTypeAlias
+    | DeclaredPredicate
+    | FunctionTypeAnnotation
+    | FunctionTypeParam
+    | IndexedAccessType
+    | IntersectionTypeAnnotation
+    | NullableTypeAnnotation
+    | ObjectTypeCallProperty
+    | ObjectTypeIndexer
+    | ObjectTypeInternalSlot
+    | ObjectTypeProperty
+    | ObjectTypeSpreadProperty
+    | OpaqueType
+    | OptionalIndexedAccessType
+    | TupleTypeAnnotation
+    | TypeAlias
+    | TypeAnnotation
+    | TypeParameter
+    | TypeParameterInstantiation
+    | TypeofTypeAnnotation
+    | UnionTypeAnnotation;
+  Super:
+    | ArrayExpression
+    | ArrowFunctionExpression
+    | AssignmentExpression
+    | AssignmentPattern
+    | AwaitExpression
```

Comment:
This is incorrect. A `Super`'s parent must be either a CallExpression or a MemberExpression. We should consider remove `Super` from `Expression` in Babel 8.